backend/src/main/java/cz/muni/fi/pa165/pneuservis/backend/entity/Customer.java

```diff
@@ -1,7 +1,179 @@
 package cz.muni.fi.pa165.pneuservis.backend.entity;
 
-import javax.persistence.Entity;
+import cz.muni.fi.pa165.pneuservis.backend.enums.VehicleTypeEnum;
 
-//@Entity
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+//import javax.validation.constraints.Size;
+import java.util.Set;
+
+/**
+ * @author Zuzana Žufanová, zufinka@mail.muni.cz
+ */
+
+@Entity
+@Table (name = "CustomerTable")

```

Comment:
I wouldn't change the name of the table if it's not necessary (e.g. for a conflict with a keyword). Or is this such case?